nesta_ds_utils/plotting.py

```diff
@@ -120,3 +123,99 @@ def save(
 
     if save_svg:
         _save_svg(fig, path, name, scale_factor, driver)
+
+
+def _find_averta() -> str:
+    """Search for averta font, otherwise return 'Helvetica' and raise a warning.
+
+    Returns:
+        str: Return averta installation name, if found. Return 'Helvetica' otherwise.
+    """
+    font = False
+    for font_path in font_manager.findSystemFonts(fontext="ttf"):
+        if "averta" in font_path:
+            font = "averta"
+            break
+        if "Averta" in font_path:
+            font = "Averta"
+            break
+    if not font:
+        warnings.warn("Averta font could not be located. Using 'Helvetica' instead")
+        font = "Helvetica"
+    return font
+
+
+def nesta_theme():
```

Comment:
missing type hint and docstring for return value

Reply:
Well spotted!